core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11971,4 +11972,25 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
 
+  @Test public void testBitNot() {
+    final RelBuilder builder = relBuilder();
+    final RexNode bitNotRexNode = builder.call(BITNOT, builder.literal(10));
+    final RelNode root = builder
+            .values(new String[]{""}, 1).project(builder
```

Comment:
could you please take the .project to the next line?

Reply:
DONE.